src/main/java/com/hms_networks/americas/sc/extensions/mqtt/MqttStatusCode.java

```diff
@@ -5,9 +5,43 @@
  *
  * @author HMS Networks, MU Americas Solution Center
  * @since 1.0.0
+ * @version 1.1.0
  */
 public class MqttStatusCode {
 
-  /** MQTT status code for 'connected' */
+  /**
+   * MQTT status code indicating that the status is unknown. This may be used to indicate that the
+   * status code is not yet known or that the status code is not available.
+   *
+   * @since 1.1.0
+   */
+  public static final int UNKNOWN = -1;
+
+  /**
+   * Internal Ewon MQTT status code for 'MQTT process start'.
+   *
+   * @since 1.1.0
+   */
+  public static final int INTERNAL_EWON_MQTT_PROCESS_START = 2;
+
+  /**
+   * MQTT status code for 'trying to connect'.
+   *
+   * @since 1.1.0
+   */
+  public static final int TRYING_TO_CONNECT = 3;
```

Comment:
```suggestion
  /**
   * MQTT status code for 'connecting'.
   *
   * @since 1.1.0
   */
  public static final int CONNECTING = 3;
```

Reply:
'Trying to connect' matches the description shown in Ewon documentation. 

I can still change this if you'd like, or do you think it should remain consistent?

Reply:
I dont think too many people are going to be concerned with the differences between official ewon documentation and our constants. I personally feel CONNECTING is more appropriate and concise. However it is debatable and fine either way. 

"TRYING_TO_CONNECT" give me the impression there is a problem during connection process and the device is still attempting the connection but has encountered something unexpected that its trying to overcome (ie. timeout, no response in given time, DNS issues, etc).

Reply:
Updated